src/Component/ProcessRunner/ProcessRunner.php

```diff
@@ -36,7 +39,9 @@ public function run(Host $host, string $command, array $config = []): string
         $defaults = [
             'timeout' => $host->get('default_timeout', 300),
             'idle_timeout' => null,
-            'cwd' => defined('DEPLOYER_ROOT') ? DEPLOYER_ROOT : null,
+            'cwd' => has('deployer_root') ? get('deployer_root') : (
```

Comment:
Lets keep only env. Right now root can be modified two ways but will affected in different places.